docs/meshstack.aks-developer-platform-quickstart-guide.md

```diff
@@ -0,0 +1,299 @@
+    1. Save this file somewhere convenient as we will also need it later.
+
+### Creating the Building Block Definition
+
+To make things easy, we will use the [meshStack Hub](https://hub.meshcloud.io) to create the new building block definition.
+
+1. Open the Github Repository Creation Definition in the hub [here](https://hub.meshcloud.io/definitions/github-repository).
+2. Click on “Add to meshStack” and enter your meshStack URL (if not given already).
+3. Continue the flow in meshStack.
+    1. Right away you will be asked for a “Type”. Pick “Workspace Building Block” here.
+    2. Continue until you reach the “Inputs” page. On this page you will be forced to fill in the following:
+        1. `GITHUB_APP_ID` - This is the value saved under “Preparation” step 1b.
+        2. `GITHUB_APP_INSTALLATION_ID` - This is the value saved under “Preparation” step 2b.
+        3. `GITHUB_APP_PEM_FILE` - This is the value saved under “Preparation” step 3a.
+4. Finish the flow and create the building block definition.
+
+🎉 The GitHub Repository Building Block Definition is ready!
+
+## 2. GitHub Actions Connector
+
+### Preparation
+
+This building block definition will interact with both GitHub & AKS, so we need authorization for both.
+
+For GitHub, you can simply reuse what we set up before.
+
+For AKS, you will need to run the Terraform code in the backplane that will set up all necessary preparation such as preparing service principals with the right permissions. You can find this Terraform code [here](https://github.com/meshcloud/meshstack-hub/blob/main/modules/aks/github-connector/backplane).
+
+Make sure you save the `config_tf` output and save it as a file called `config.tf`. We will use it later.
+
+### Creating the Building Block Definition
+
+We will again use the [meshStack Hub](https://hub.meshcloud.io) to create the new building block definition.
+
+1. Open the Github Actions Integration with AKS definition in the hub [here](https://hub.meshcloud.io/definitions/aks-github-connector).
+2. Click on “Add to meshStack” and enter your meshStack URL (if not given already).
+3. Continue the flow in meshStack.
+    1. Right away you will be asked for a “Type”. Pick “Tenant Building Block” here and select “AKS” as the supported platform.
+    2. Continue until you reach the “Dependencies” page. On this page, select the “Github Repository Creation” building block definition that you created before.
+    3. Continue until you reach the “Inputs” page. On this page you will be forced to fill in the following:
+        1. `namespace` - This refers to the AKS namespace. Open this up and set the “Source” to “Platform Tenant ID”.
+        2. `github_repo` - This refers to the GitHub repository to connect to. Open this up and set the “Source” to “Parent Building Block Output” and set the corresponding output to “repo_name”.
+        3. `GITHUB_APP_ID` - This is the value saved under “Preparation” step 1b from the previous building block definition
```

Comment:
Most of the things were provided previously

Reply:
And you have to do this again :-)